data/wanderer/wanderer ships.txt

```diff
@@ -259,6 +259,64 @@ ship "Strong Wind"
 
 
 
+ship "Heavy Gust"
+	sprite "ship/heavy gust"
+	thumbnail "thumbnail/heavy gust"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Wanderer Military"
+		"cost" 17600000
+		"shields" 35200
+		"hull" 21200
+		"required crew" 22
+		"bunks" 34
+		"mass" 652
+		"drag" 7.4
+		"heat dissipation" .45
+		"fuel capacity" 600
+		"cargo space" 52
+		"outfit space" 534
+		"weapon capacity" 224
+		"engine capacity" 132
+		"tactical scan power" 50
```

Comment:
This could probably be boosted a bit, as you're losing net "implicit" outfit space with the scanning changes, without adding that gain to the overall "outfit space" (the overall outfit space increase feels a tad on the low side to me, but well within reason). Adding another 50% or more tactical scan power still has the scanning change lose net "implicit" outfit space, but feels like a more reasonable tradeoff. Especially as that's still weaker tactical scanning than a 7 outfit space Remnant "Salvage Scanner" that has 84 tactical scan power and other uses, and significantly weaker than a 12 outfit space Avgi "Multispectral Scanner" with 180 tactical scan power (though with the caveat that the Avgi one comes with more significant drawbacks).
```suggestion
		"tactical scan power" 75
```